utils.groovy

```diff
@@ -194,6 +194,18 @@ def tryWithMessagingCredentials(Closure body) {
     }
 }
 
+// Send slack message if the credential exists. Do nothing if not.
+def trySlackSend(color, message) {
```

Comment:
What does this get us? All the Slack sends are gated on `official` already. Keying off of the credential existing makes sense, but as a complexity trade I'd remove the `official` gates in that case.

Reply:
Maybe simpler we skip that commit for now and circle back to it?

Reply:
yeah. The desire is to remove `official` in the future. I can drop that commit.

Reply:
removed that 2nd commit